openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -48,6 +49,8 @@ window.Browser = Browser;
 window.Subject = Subject;
 window.Template = Template;
 
+window.onload = initializeSwaggerUI;
```

Comment:
Lets follow the same pattern you did in your last PR:
https://github.com/internetarchive/openlibrary/pull/8447/files#diff-587fb865c85bb3325504c795a40db189d2e702d42d9d160304607e027c142ebdR543-R549

I think that may also fix the reason the tests are failing (which is the very large all.js file size) https://github.com/internetarchive/openlibrary/actions/runs/6676008459/job/18149274532

Reply:
@imperial-chief the reason we don't want to do window.onload is because that would run for every page on openlibrary, not just the swagger ui page.

Reply:
> Lets follow the same pattern you did in your last PR: https://github.com/internetarchive/openlibrary/pull/8447/files#diff-587fb865c85bb3325504c795a40db189d2e702d42d9d160304607e027c142ebdR543-R549
> 
> I think that may also fix the reason the tests are failing (which is the very large all.js file size) https://github.com/internetarchive/openlibrary/actions/runs/6676008459/job/18149274532

Well but in the previous case, the javascript used to work on selected elements but in this case, it works on complete window. That's why i did this.

Reply:
> Lets follow the same pattern you did in your last PR: https://github.com/internetarchive/openlibrary/pull/8447/files#diff-587fb865c85bb3325504c795a40db189d2e702d42d9d160304607e027c142ebdR543-R549
> 
> I think that may also fix the reason the tests are failing (which is the very large all.js file size) https://github.com/internetarchive/openlibrary/actions/runs/6676008459/job/18149274532

Okay let me try.

Also, Can you tell me which web page does this code affects so that i can if its working or not.

Reply:
https://openlibrary.org/swagger/docs

So the and the query selector should be #swagger-ui because that's the id of the element that's on the page.